app/controllers/submit.js

```diff
@@ -854,10 +847,10 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                 .map(item => item.trim());
 
             if (this.get('abstractChanged'))
-                node.set('description', this.get('basicsAbstract'));
+                model.set('description', this.get('basicsAbstract'));
 
             if (this.get('tagsChanged'))
-                node.set('tags', this.get('basicsTags'));
+                model.set('tags', this.get('basicsTags'));
 
             if (this.get('applyLicense')) {
                 if (node.get('nodeLicense.year') !== this.get('basicsLicense.year') || (node.get('nodeLicense.copyrightHolders') || []).join() !== copyrightHolders.join()) {
```

Comment:
Should this be looking at `model.get('licenseRecord.year')` and `model.get('licenseRecord.copyrightHolders')`?

Reply:
for this PR, at this point in the divorce, we are holding off removing license related node calls.

Reply:
:+1: